### PR TITLE
Adapt ND to changes in code (bug/feature?) where 0 means no limit

### PR DESCRIPTION
**Pull request by mwoz123**
Adapt ND to changes in code (bug/feature?) where 0 means no limit (instead of previous "no download") + reorganize config, improved comments

### CODE REVIEW

**Comment by Frenzie** [comment]
That was an explicit change in #3072. 

> Fixes unlimited (limit = 0) was broken

I thought the fact that 0 didn't function as "no limit" was an unmistakable bug. I still would've advocated for its change if I realized that it was intended that way. I don't think I kept that change a secret or anything. :-P

File: plugins/newsdownloader.koplugin/feed_config.lua [hunk 1]
```diff
@@ -1,8 +1,13 @@
 return {
 	-- list your feeds here:
+
 	-- only supports http URL for now
```
Comment (Frenzie) [inline plugins/newsdownloader.koplugin/feed_config.lua]: This is no longer true.

**Comment by mwoz123** [comment]
Not sure if it's a bug or feature ;P

I first code limit = "0" to equal no download, "-1" no limit. 
But since some else changed (or bugged;)  it I think it makes more sense to have 0 equal to unlimited, as you can always comment out what you want to remove temporarily (disable). 

Are you sure 0 again means no download? 
Yesterday I downloaded koreader C.I. Android version 1152 as I it was downloading all default football news

& so I adapted configuration...

**Comment by Frenzie** [comment]
Right, I *thought* it was broken because 0 is normally supposed to mean no limit. -1 makes more sense as disabled.

**Pull request merged**